src/test/java/edu/oakland/test/display02/DisplayCommManagerImplementationTests.java

```diff
@@ -0,0 +1,40 @@
+package edu.oakland.test.display02;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.display02.DisplayCommManager;
+import edu.oakland.production.display02.DisplayCommManagerImplementation;
+import edu.oakland.production.middleware02.MiddlewareCommInterface;
+import edu.oakland.test.display02.MiddlewareCommInterfaceStub;
+import java.lang.IllegalArgumentException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Display Comm Manager Implementation Tests")

```

Comment:
For sake of standardization to make it easier to read and search the logs, can we change this to `DisplayCommManagerImplementation Unit Tests`? Thank you!
```suggestion
@DisplayName("DisplayCommManagerImplementation Unit Tests")

```